pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1004,7 +1005,7 @@ def _check_consider_using_generator(self, node):
                 # remove square brackets '[]'
                 inside_comp = node.args[0].as_string()[1:-1]
                 call_name = node.func.name
-                if call_name in {"any", "all", "sum", "max", "min"}:
+                if call_name in {"any", "all"}:
```

Comment:
Is this the only change necessary for this to work? That's nice 😄

Reply:
Haha yeah, thanks to great code by @ikraduya :)